tools/buildmgr/cbuildgen/src/CMakeListsGenerator.cpp

```diff
@@ -223,67 +226,186 @@ bool CMakeListsGenerator::GenBuildCMakeLists(void) {
     }
   }
 
-  // File specific flags
```

Comment:
Maybe it would be shorter to implement the code below similarly to file/group specific defines? Please see the loops between lines 69 and 104.

Reply:
this should be addressed into separate PR